### PR TITLE
 Apr 22, Friday morning: Whats-new: Removal of estimated cost

### PR DESCRIPTION
**Pull request by jfl920**
@zuluecho9 will be handling this Friday morning. Coinciding with this https://github.com/newrelic/docs-website/pull/7173.

### CODE REVIEW

**Comment by zuluecho9** [comment]
@aalfano7

**Review by aalfano7 — COMMENTED** [review]
I suggest the following:

---
title: Simplified usage and cost tracking
summary: We've simplified your end-of-month costs.
releaseDate: '2022-04-02'
----
## Changes to the usage and cost breakdowns in your account administration area

In our efforts to improve the accuracy and clarity of data for our customers, we have updated your [Usage and cost overview](https://one.newrelic.com/usage-and-cost) to no longer include monthly estimated costs, effective April 22, 2022. We are making this change to avoid confusion in scenarios where estimated costs may differ from actual month-to-date costs, which can happen for a variety of reasons. Your usage and cost data will continue to display user counts and gigabytes ingested within the current month. Your monthly invoice provides an accurate, single source of truth regarding monthly costs.

## How to view your costs
If you did not purchase New Relic through a third party and you are authorized to manage billing, you can access your most recent invoices by navigating to [Manage your plan](
https://one.newrelic.com/nr1-core/plan-management/home) under the account administration dropdown. This is your source of truth for billing information. 

Additional information about our pricing and billing UI can be found [here](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-pricing-billing/billing-ui/). 

Note that you can still calculate and view your estimated month-to-date costs via NRQL queries. Reference our [docs](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#estimated-cost-removal) for more information.

File: src/content/whats-new/2022/04/whats-new-4-20-removal-of-estimated-costs.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: Simplified usage and cost tracking
+summary: We've simplified your end-of-month costs.
```
Comment (jfl920) [inline src/content/whats-new/2022/04/whats-new-4-20-removal-of-estimated-costs.md:3]: Given that this announcement is going out before the actual change - can we change this to "We're simplifying .."

File: src/content/whats-new/2022/04/whats-new-4-20-removal-of-estimated-costs.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+releaseDate: '2022-04-22'
+----
+## Changes to the usage and cost breakdowns in your account administration area
+
+In our efforts to improve the accuracy and clarity of data for customers on our usage-based New Relic One pricing model, we've updated your [Usage and cost overview](https://one.newrelic.com/usage-and-cost) to no longer include monthly estimated costs, effective April 22, 2022. We are making this change to avoid confusion in scenarios where estimated costs may differ from actual month-to-date costs, which can happen for a variety of reasons. Your usage and cost data will continue to display user counts and gigabytes ingested within the current month. Your monthly invoice provides an accurate single source of truth for monthly costs.
```
Comment (jfl920) [inline src/content/whats-new/2022/04/whats-new-4-20-removal-of-estimated-costs.md:8]: effective May 9th, 2022

**Pull request merged**